frame/bridge/ethereum/backing/src/lib.rs

```diff
@@ -100,6 +100,7 @@ mod types {
 
 	pub type RingBalance<T> =
 		<<T as Trait>::RingCurrency as Currency<<T as frame_system::Trait>::AccountId>>::Balance;
+	#[cfg(feature = "std")]
```

Comment:
Why need to add this?

Reply:
Just suppress the `unused import` warning in no std.